lib/slack.ex

```diff
@@ -171,6 +171,20 @@ defmodule Slack do
   end
 
   @doc """
+  Sends `text` and `attachments` to `channel` for the given `slack` connection.
```

Comment:
Link to `https://api.slack.com/docs/attachments` and mention how to format attachments?